themes/finna2/templates/RecordTab/usercomments.phtml

```diff
@@ -4,55 +4,62 @@
   $user = $this->auth()->isLoggedIn();
   $ratings = $this->record($this->driver)->ratingAllowed();
   $allowCommenting = $user && $this->record($this->driver)->commentingAllowed($user);
+  $recordType = $this->driver->tryMethod('getRecordType');
 ?>
-<div class="comment-list">
-  <?=$this->render('record/comments-list.phtml', ['driver' => $this->driver])?>
-</div>
+<?php if ($recordType === 'lrmi' && $externalLink = $this->driver->getExternalLink()): ?>
+  <p class="alert alert-aoe">
+    <?= $this->transEsc('add_aoe_rating') ?> (<a href="<?=$externalLink?>">aoe.fi</a>).
+  </p>
```

Comment:
Tähän voisi laittaa return:n, niin ei tarvitse koko lopputemplaattia laittaa else-haaraan.

Reply:
Iso `else`-haara on ikävä, mutta enemmän vierastan `return`:in käyttöä templateissa. Tällaisessa tapauksessa lähtisin ennemmin miettimään useampaan templateen jakamista. Harmi, että ajurikohtaisille RecordTab-templateille ei näyttäisi olevan suoraan tukea.

Reply:
return templaten alussa on ihan ok. returnit keskellä tai siellä täällä ovat sekavia, mutta jos heti aluksi todetaan, ettei tarvita koko templatea, niin on lupa poistua. Jos koet vahvasti tämän, niin ei ole kynnyskysymys, mutta vahva suositus silti.

Reply:
Saa toki tehdä alitemplaatillakin. Ja tämä ei ole Twig eikä mikään muukaan, joten voi tehdä asioita myös eri tavalla.

Reply:
Laitoin nyt early exitin kun muuttuneen toteutuksen myötä sai hoitumaan heti alun PHP-blokissa. Nuo returnit aiheuttavat kyllä vähän haastetta templaattien alku-/loppukommenttienkin kanssa. Loppukommentin toistaminen tuntui hölmöltä, joten ratkaisin siirtämällä alkukommenttia.